content/index.md

```diff
@@ -76,6 +76,11 @@ The Multiformats project began through [the IPFS Project](https://ipfs.io). It i
 		<img height="64px" src="https://github.com/libra/libra/raw/master/.assets/libra.png" />Libra</a>
 		- Libra’s mission is to enable a simple global currency and financial infrastructure that empowers billions of people.
 	</li>
+	<li>
+		<a href="https://1.bp.blogspot.com/-0SArWfduw68/XkxV8EmBBcI/AAAAAAAAABw/h9aWSWbm0J4kilgn3xddzQ3PdoP-e3RZgCLcBGAsYHQ/s1600/SAVE_20200127_132431.jpg">
```

Comment:
I think this should be linking the project, not the image?

Reply:
Thank you. Will update.

Reply:
https://github.com/NamChain-Open-Initiative-Research-Lab/NamChain is the link of the project.

Reply:
Can you update the link in the PR

Reply:
Updated.